config/grad.uiowa.edu/core.entity_view_display.node.person.teaser.yml

```diff
@@ -75,13 +83,52 @@ content:
     settings:
       link_to_entity: false
     third_party_settings: {  }
-  field_teaser:
-    type: basic_string
-    weight: 4
+  field_person_scholar_abstract:
+    type: smart_trim
+    weight: 8
     region: content
-    label: hidden
-    settings: {  }
+    label: visually_hidden
+    settings:
+      trim_length: 100
+      trim_type: words
+      trim_suffix: ''
+      wrap_class: trimmed
+      more_link: true
+      more_text: More
+      more_class: grad-more-link
```

Comment:
I'm assuming this is just allowing us to style this as a button? Can we just add existing button classes here?

Reply:
Since the view hasn't been created yet, I just removed the more link for now. Thank you for pointing that out!